grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -3698,21 +3698,25 @@ def section_memory_manager(outer_panels):
                     ],
                 ),
                 panels.timeseries_count(
```

Comment:
```suggestion
                panels.timeseries(
```

See the differences in https://github.com/risingwavelabs/risingwave/blob/20fc87999a65c831e24006fff893604589e49547/grafana/common.py#L157-L178